Helper/Data.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Copyright ©  All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace HappyHorizon\PersistentGraphQlSchema\Helper;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\Exception\FileSystemException;
+use Magento\Framework\Filesystem\Driver\File;
+use Magento\Framework\Filesystem\DirectoryList;
+use Magento\Backend\Controller\Adminhtml\Cache\MassRefresh;
+
+class Data extends AbstractHelper
+{
+    /**
+     * @param Context $context
+     * @param DirectoryList $dir
+     * @param File $file
+     * @param MassRefresh $massRefresh
+     */
+    public function __construct(
+        protected Context $context,
+        protected DirectoryList $dir,
+        protected File $file,
+        protected MassRefresh $massRefresh
+    ) {
+        parent::__construct($context);
+    }
+
+    /**
+     * @return string
+     * @throws FileSystemException
+     */
+    public function getGqlPath()
+    {
+        return $this->dir->getPath('etc').'/gql.php';
```

Comment:
There are now 2 different functions with their own string for the file name/location

Since this is a public function of the helper, please make the plugin use this function to determine file name/location

(ideally, the 'gql.php' should be a const)